### PR TITLE
Basic styling with elm-css 💄 💅 

### PR DESCRIPTION
**Pull request by charypar**
This adds basic styling for the UI, using [elm-css](https://github.com/rtfeldman/elm-css). I tried to still support building with Webpack with HMR and managed to do it, but at the cost of not having webpack config in ES6.

On top of that, I needed to split the webpack config in order to support only running extract text plugin for static build (and not while running in dev with HMR).

Part of #12.

# to do

- [x] work out the configuration to make elm-css and webpack work 
- [ ] ~~figure out why the hot config sometime crashes and sometimes works fine (wtf.)~~
- [x] put styles next to UI components
- [x] basic styling for the main UI
- [ ] ~~investigate using bootstrap/foundation with elm-css (may not be very useful)~~
- [x] basic styling for HAR components (stripy table with reasonable paddings, ...)

### CODE REVIEW

File: app/ui/webpack.config.hot.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+var config = require('./webpack.config.base');
```
Comment (jameshopkins) [inline app/ui/webpack.config.hot.js]: `const`

**Comment by charypar** [comment]
Weirdly it seems like the webpack config is broken up until the first time it does HMR, from then on it works ok... I'm confused.

File: app/ui/webpack.config.hot.js (hunk 1 above)
Comment (lpil) [inline app/ui/webpack.config.hot.js]: Surely you're using a new enough version of node to use `let`, it's been around since ~v0.10

Comment (jameshopkins) [inline app/ui/webpack.config.hot.js]: As @charypar rightly pointed out, it's a WIP. Can go back later and change it

Comment (charypar) [inline app/ui/webpack.config.hot.js]: What I meant, @jameshopkins, is the value is mutable (so can't/shouldn't be a `const`) , but it totally can be a `let`, although I can't help feeling this level of focus on a config file is a bit OCD 😄

**Comment by charypar** [comment]
httpmark

**Comment by jameshopkins** [comment]
> Weirdly it seems like the webpack config is broken up until the first time it does HMR, from then on it works ok... I'm confused.

Is this still the case?

**Comment by charypar** [comment]
@jameshopkins Doesn't happen to me any more, you?

**Comment by jameshopkins** [comment]
> Doesn't happen to me any more, you?

Admittedly, I haven't checked out the branch myself. Just asking since there's been no subsequent commits since [your comment](https://github.com/httpmark/httpmark/pull/58#issuecomment-266314888) that indicated that the issue had been fixed.

OK for me to merge then?

**Comment by charypar** [comment]
Yeah, go ahead, if it comes back, we'll treat is as an issue.

**Pull request merged**